scripts/create_metadata

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+# OUT_DIR is where the file gets written.
+# The main purpose of this is to make testing easier.
+OUT_DIR="${OUT_DIR:-.}"
+
+# Check if DEV_TAGS is unset, if not, set it to empty.
+if [ -z "${DEV_TAGS:-}" ]; then
```

Comment:
Nit: you could also represent this similar to `OUT_DIR` as `${DEV_TAGS:=""}` with [parameter expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html)